File: c++/Lv1/두 정수 사이의 합.cpp
#include <string>
#include <vector>

using namespace std;

long long solution(int a, int b) {
    long long answer = 0;
    int k=b>a?b-a+1:a-b+1;
    answer=(long long)(a+b)*k/2;
    return answer;
}

File: Python/Programmers/Lv1/두 정수 사이의 합.py
#https://school.programmers.co.kr/learn/courses/30/lessons/12912

def solution(a, b):
    if b>a:
        answer = (a+b)*(b-a+1)/2
    else:
        answer = (a+b)*(a-b+1)/2
    return answer